quiz_hosting_web_app/quiz_management/decorators.py

```diff
@@ -0,0 +1,23 @@
+from django.http import HttpResponse
+from .models import QuizAttempter
+
+
+def host_required(view_function):
+    def check_host(request, *args, **kwargs):
+        if request.user.is_authenticated and not request.user.is_superuser:
+            try:
+                is_quiz_attempter = QuizAttempter.objects.get(id=request.user.id).is_quiz_attempter
+                if not is_quiz_attempter:
+                    return view_function(request, *args, **kwargs)
+            except QuizAttempter.DoesNotExist:
+                return view_function(request, *args, **kwargs)
+        return HttpResponse("You are not Host/")
```

Comment:
Instead of returning HTTPResponse, can we redirect the user to his homepage instead of displaying this page?